docs/installation/vagrant-boxes.md

```diff
@@ -0,0 +1,28 @@
+---
+title: 'Vagrant Boxes'
+---
+
+###### last updated: 2024-07-02
+
+# Vagrant Boxes
+
+AlmaLinux offers official AlmaLinux OS images for Vagrant Boxes. Images are delivered by the AlmaLinux [Cloud SIG](https://wiki.almalinux.org/sigs/Cloud.html).
+
+The following [AlmaLinux OS 9](https://app.vagrantup.com/almalinux/boxes/9) and [AlmaLinux OS 8](https://app.vagrantup.com/almalinux/boxes/8) images are available on Vagrant Cloud:
+| Vagrant Box| AlmaLinux OS 9 | AlmaLinux OS 8 |
+| --- | --- | --- |
+| Libvirt | Intel/AMD (x86_64) | Intel/AMD (x86_64) | 
+| Virtual Box | Intel/AMD (x86_64) | Intel/AMD (x86_64) |
+| Hyper-V | Intel/AMD (x86_64) | - |
+| VMWare | Intel/AMD (x86_64)<br>ARM64 (Aarch64) | Intel/AMD (x86_64) |
+| Parallels | ARM64 (Aarch64) | - | 
+
+## Changelog
+
+### AlmaLinux OS  version **8.10.20240604**
```

Comment:
Please, remove this part. Even image supports Vagrant doesn't support to change it on the VM creation.